src/MultiSigInbox/MultiSigInboxPlugin.cs

```diff
@@ -0,0 +1,191 @@
+            }
+            else
+            {
+                Console.WriteLine("Transaction was not found");
+            }
+        }
+
+        [ConsoleCommand("inbox send", Category = "MultiSigInbox", Description = "Send transaction")]
+        private void OnInboxSend(string transaction)
+        {
+            ContractParametersContext context = ContractParametersContext.FromJson(JObject.Parse(transaction), System.StoreView);
+
+            var tx = store.TryGet(TxPrefix.Concat(context.Verifiable.Hash.ToArray()).ToArray());
+            if (tx != null)
+            {
+                Console.WriteLine("Transaction already sent");
+                return;
+            }
+
+            store.Put(TxPrefix.Concat(context.Verifiable.Hash.ToArray()).ToArray(), Utility.StrictUTF8.GetBytes(transaction));
+
+            using var snapshot = System.GetSnapshot();
+            RelayContext(snapshot, context);
+        }
+
+        [ConsoleCommand("inbox sign", Category = "MultiSigInbox", Description = "Sign transaction")]
+        private void OnInboxSign(UInt256 txHash)
+        {
+            var tx = store.TryGet(TxPrefix.Concat(txHash.ToArray()).ToArray());
+            if (tx != null)
+            {
+                using var snapshot = System.GetSnapshot();
+                var context = ContractParametersContext.FromJson(JObject.Parse(tx), snapshot);
+
+                Console.WriteLine(tx.AsSerializable<Transaction>().ToJson(System.Settings).ToString(true));
+                RelayContext(snapshot, context);
+            }
+            else
+            {
+                Console.WriteLine("Transaction was not found");
+            }
+        }
+
+        private void RelayContext(SnapshotCache snapshot, ContractParametersContext context)
+        {
+            if (context.Completed)
+            {
+                Log($"Send tx: hash={context.Verifiable.Hash}");
+                context.Verifiable.Witnesses = context.GetWitnesses();
+                System.Blockchain.Tell(context.Verifiable);
+            }
+            else
+            {
+                var tx = store.TryGet(TxPrefix.Concat(context.Verifiable.Hash.ToArray()).ToArray());
+
+                foreach (var wallet in _wallet.GetAccounts())
+                {
+                    var msg = new ExtensiblePayload()
+                    {
+                        Category = MutiSigPayloadCategory,
+                        Data = tx,
+                        ValidBlockStart = 0,
+                        ValidBlockEnd = NativeContract.Ledger.CurrentIndex(snapshot) + 1_000,
+                        Sender = wallet.ScriptHash
+                    };
+
+                    var sign = new ContractParametersContext(snapshot, msg);
+                    if (_wallet.Sign(sign) && sign.Completed)
+                    {
+                        msg.Witness = sign.GetWitnesses()[0];
+                        System.Blockchain.Tell(sign.Verifiable);
+                        break;
```

Comment:
It's better to show txid, otherwise if many txids in inbox list, I don't know which is mine, which need to sign.
```suggestion
                        System.Blockchain.Tell(sign.Verifiable);
                        Console.WriteLine($"TXID: {context.Verifiable.Hash}");
                        break;
```